src/SymfonyCache/PurgeListener.php

```diff
@@ -81,7 +93,20 @@ public function handlePurge(CacheEvent $event)
         }
 
         $response = new Response();
-        if ($event->getKernel()->getStore()->purge($request->getUri())) {
+        $store = $event->getKernel()->getStore();
+
+        if ($request->headers->has($this->purgeTagsHeader)
+            && $store instanceof TaggableStore
```

Comment:
should we not fail if there is a request to purge tags but we don't have a store supporting purge?

Reply:
Matter of preference. I thought about that too. Either fail or ignore silently. You prefer failing?

Reply:
its a basic setup error, not a runtime error that could occasionally happen, so yes i would explode loudly.

Reply:
Implemented with the new listener now.